README.md

```diff
@@ -83,6 +83,7 @@ List of every HackerOoT contributors, from most recent to oldest contribution:
 
 - hiisuya
 - Zeldaboy14
+- Sauraen
```

Comment:
this list is supposed to be ordered from newest to oldest btw

Reply:
yeah I corrected that, sauraen contributed before and was never credited here that's why the location might seem odd